File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/ProcessingJobUpdateJobTypesRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;import java.util.List;
import com.fasterxml.jackson.annotation.JsonProperty;public class ProcessingJobUpdateJobTypesRequestItem {
    @JsonProperty("Attributes")
    public List<Object> attributes;
    @JsonProperty("CategoryName")
    public String categoryName;
    @JsonProperty("Description")
    public String description;
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("Name")
    public String name;
    @JsonProperty("ProcessingSteps")
    public String processingSteps;
}

File: sdks/thunkmetrc-go/wrapper/models/processing_job_update_job_types_request_item.go
package models

type ProcessingJobUpdateJobTypesRequestItem struct {
    Attributes []string `json:"Attributes,omitempty"`
    CategoryName string `json:"CategoryName,omitempty"`
    Description string `json:"Description,omitempty"`
    Id int `json:"Id,omitempty"`
    Name string `json:"Name,omitempty"`
    ProcessingSteps string `json:"ProcessingSteps,omitempty"`
}
